doc/make.py

```diff
@@ -302,6 +306,12 @@ def main():
     argparser.add_argument(
         "--no-api", default=False, help="omit api and autosummary", action="store_true"
     )
+    argparser.add_argument(
+        "--no-ipython",
+        default=False,
```

Comment:
is `default` needed with `action="store_true"`?

Reply:
Not sure; I just copied from `--no-api` above.

Reply:
Yeah I think Marco is correct - https://docs.python.org/3/library/argparse.html#action